release/cli/pkg/helm/helm.go

```diff
@@ -466,19 +466,17 @@ func OverWriteChartValuesImageSha(filename string, shaMap map[string]anywherev1a
 	return nil
 }
 
-func GetPackagesImageTags(eksArtifacts map[string][]releasetypes.Artifact) (map[string]string, error) {
+func GetPackagesImageTags(packagesArtifacts map[string][]releasetypes.Artifact) (map[string]string, error) {
 	m := make(map[string]string)
-	for _, artifacts := range eksArtifacts {
+	for _, artifacts := range packagesArtifacts {
 		for _, artifact := range artifacts {
 			if artifact.Image != nil {
-				if artifact.Image.AssetName == "eks-anywhere-packages" || artifact.Image.AssetName == "ecr-token-refresher" {
-					m[artifact.Image.AssetName] = artifact.Image.ReleaseImageURI
-				}
+				m[artifact.Image.AssetName] = artifact.Image.ReleaseImageURI
 			}
 		}
 	}
 	if len(m) == 0 {
-		return nil, fmt.Errorf("No assets found for eks-anywhere-packages, or ecr-token-refresher in eksArtifacts")
+		return nil, fmt.Errorf("No assets found for eks-anywhere-packages, or ecr-token-refresher in packagesArtifacts")
```

Comment:
Should this just be `eks-anywhere-packages`? Assuming I'm understanding that ecr-token-refresher was removed from the artifacts earlier.

Reply:
The check was
```go
if artifact.Image.AssetName == "eks-anywhere-packages" || artifact.Image.AssetName == "ecr-token-refresher"
```
I only removed the mapping of `ecr-token-refresher` project name to a list of artifacts, since we don't have a project like that. The image artifact for ecr-token-refresher very much exists, inside the artifacts list mapped to project name `eks-anywhere-packages`, and that's what we're checking for here.